br/pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -972,33 +972,39 @@ func NewFileCheckpointsDB(ctx context.Context, path string) (*FileCheckpointsDB,
 	cpdb.fileName = fileName
 	cpdb.exStorage = s
 
-	// ignore all errors -- file maybe not created yet (and it is fine).
-	content, err := cpdb.exStorage.ReadFile(ctx, fileName)
-	if err == nil {
-		err2 := cpdb.checkpoints.Unmarshal(content)
-		if err2 != nil {
-			log.L().Error("checkpoint file is broken", zap.String("path", path), zap.Error(err2))
-		}
-		// FIXME: patch for empty map may need initialize manually, because currently
-		// FIXME: a map of zero size -> marshall -> unmarshall -> become nil, see checkpoint_test.go
-		if cpdb.checkpoints.Checkpoints == nil {
-			cpdb.checkpoints.Checkpoints = map[string]*checkpointspb.TableCheckpointModel{}
-		}
-		for _, table := range cpdb.checkpoints.Checkpoints {
-			if table.Engines == nil {
-				table.Engines = map[int32]*checkpointspb.EngineCheckpointModel{}
-			}
-			for _, engine := range table.Engines {
-				if engine.Chunks == nil {
-					engine.Chunks = map[string]*checkpointspb.ChunkCheckpointModel{}
-				}
-			}
-		}
-	} else {
+	isExists, err := cpdb.exStorage.FileExists(ctx, fileName)
```

Comment:
Nit: `exist` is actually a verb. 🤣

Reply:
may be i can change to `exist`, like https://github.com/pingcap/tidb/blob/master/br/pkg/backup/client.go#L176.